src/lsm/schema.zig

```diff
@@ -334,6 +335,65 @@ pub const TableData = struct {
     }
 };
 
+pub const FreeSetNode = struct {
+    pub const Word = u64;
```

Comment:
This seemingly duplicates `FreeSet.Word`, but I think that's false sharing --- there's no _fudnamental_ reason when in-memory and on-disk words must match. We require it in the current implementation, and indeed, in encoded free set, we assert that the two are the same.